x-pack/plugins/ml/public/application/settings/anomaly_detection_settings.tsx

```diff
@@ -160,7 +165,79 @@ export const AnomalyDetectionSettings: FC = () => {
             </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlexItem>
-        <EuiFlexItem grow={5}>
+
+        <EuiFlexItem>
+          <EuiTitle size="s">
+            <h3>
+              <FormattedMessage
+                id="xpack.ml.settings.anomalyDetection.calendarsDstTitle"
+                defaultMessage="Calendars DST"
+              />
+            </h3>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+          <EuiText size="s">
+            <EuiTextColor color="subdued">
+              <p>
+                <FormattedMessage
+                  id="xpack.ml.settings.anomalyDetection.calendarsDstText"
+                  defaultMessage="DST Calendars contain a list of scheduled events for which you do not want to generate anomalies, taking into account daylight saving time shifts that may cause events to occur one hour earlier or later."
```

Comment:
Nit - sentence case `DST calendars` better here?

Reply:
Updated in https://github.com/elastic/kibana/pull/193605/commits/db72ef0440d4e606a284040d561af0f981647fc5